incubator/bootstrap_cli/__main__.py

```diff
@@ -826,16 +857,22 @@ def prepare(self):
             {"projectsAcl": {"actions": ["READ", "UPDATE"], "scope": {"all": {}}}},
         ]
 
-        group_create_object = Group(name=group_name, capabilities=group_capabilities)
-        if self.config.cognite.idp_authentication:
-            # inject (both will be pushed through the API call!)
-            group_create_object.source_id = self.config.cognite.idp_authentication.client_id
-            # type: ignore # 'AD Group FooBar' # type: ignore
-            group_create_object.source = f"AAD Server Application: {group_create_object.source_id}"
-        self.client.iam.groups.create(group_create_object)
+        # TODO: replace with dataclass
+        aad_mapping = [
+            # sourceId
+            (source_id := self.config.cognite.idp_authentication.client_id),
```

Comment:
why is this done? I had to create a dedicated AAD-mapping for this to work. IE create a new group, add the service principal and update the mapping in the ui.

Reply:
this issue is resolved now